src/layout/CustomButton/config.ts

```diff
@@ -0,0 +1,50 @@
+import { CG } from 'src/codegen/CG';
+import { CompCategory } from 'src/layout/common';
+
+export const Config = new CG.component({
+  category: CompCategory.Action,
+  rendersWithLabel: false,
+  capabilities: {
+    renderInTable: false,
```

Comment:
I think it should be possible to render this in a table (i.e. Grid). Don't quite see why that wouldn't be useful? :thinking: